segmentation/cells/UNet3DPlatyCellProbs.model/UNet3DPlatyCellProbs.model.yaml

```diff
@@ -0,0 +1,61 @@
+name: 3D UNet Platynereis Cell Segmentation (Probabilities)
+description: A 3d U-Net trained to predict the cell boundaries in a EM volume of a 6 day old Platynereis.
+cite:
+  - text: Vergara, Hernando M. et al. Whole-body integration of gene expression and single-cell morphology. BioRxiv 2020."
+    doi: https://doi.org/10.1101/2020.02.26.961037
+authors:
+  - Constantin Pape;@bioimage-io
+documentation: ../../README.md
+tags: [unet3d, pytorch, platynereis, cell membrane, segmentation, EM]
+license: MIT
+
+format_version: 0.1.0
+language: python
+framework: pytorch
+
+source: mmpb.segmentation.network.models.UNetAnisotropic
+optional_kwargs:
+  in_channels: 1
+  out_channels: 6
```

Comment:
This should be 1, because we will average over the affinities (= channel axis)

Reply:
no it should be just like the model you trained as it's the (not so optional) kwargs for UNetAnisotropic class.

Reply:
Ah makes sense.
Then it's 12 though.
(It's a bit confusing, the value 6 from the config file is not used, because this is set dynamically in the code from the number of affinity channels).